Bookish/Bookish.Web/Models/BookViewModel.cs

```diff
@@ -14,5 +10,17 @@ public BookViewModel(Book book)
         {
             Book = book;
         }
+
+        public string IsDisabled(BookCopy copy)
+        {
+            return copy.username != null ? "disabled" : "";
```

Comment:
I would split out the `copy.username != null` check into a separate `IsAvailable` check, just for readability. You can then reuse it in the method below 🙂